lib/msal-browser/src/telemetry/BrowserPerformanceMeasurement.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IPerformanceMeasurement } from "@azure/msal-common";
+
+export class BrowserPerformanceMeasurement implements IPerformanceMeasurement {
+    private measureName: string;
+    private correlationId: string;
+    private startMark: string;
+    private endMark: string;
+
+    constructor(name: string, correlationId?: string) {
```

Comment:
Should we just make correlationId required?

Reply:
Yes, decided to take this approach and generate a placeholder correlation id if none is provided.